views/pages/searchProjectsByTag.ejs

```diff
@@ -0,0 +1,190 @@
+                        Delete
+                    </button>
+                </div>
+            </div>
+
+            <% } %>
+            <table id="results"></table>
+        </div>
+    </div>
+
+
+</div>
+
+<% include ../partials/footer.ejs %>
+
+<script src="/lib/bootstrap-table/bootstrap-table.min.js"></script>
+<script src="/stickyTablePageSize.js"></script>
+<script>
+
+    
+    $table = $('#results');
+
+    $table.bootstrapTable({
+        method: 'get',
+        url: '/api/projects/tag/<%= tag.id %>',
+        cache: false,
+        striped: true,
+        escape: true,
+        idField: 'id',
+        search: true,
+        sortName: 'name',
+        showColumns: true,
+        showExport: true,
+        showRefresh: true,
+        searchOnEnterKey: true,
+        showPaginationSwitch: true,
+        pagination: true,
+        pageSize: page_size, // declared and set in stickyTablePageSize.js 
+        // reach the columns nested in the response data
+        responseHandler: function(res) {
+            return res.result; 
+        },
+        columns: [
+        <% if(user.admin) { %>
+            {
+                field: 'state',
+                checkbox: 'true'
+            }, 
+        <% } %>
+        {
+            field: 'id',
+            title: 'ID',
+            visible: false
+        }, {
+            field: 'name',
+            title: 'Name',
+            sortable: true,
+            formatter: operateFormatter
+        }, {
+            field: 'description',
+            title: 'Description',
+            sortable: false
+        }, {
+            field: 'tags',
+            title: 'Tags',
+            sortable: true
+        }]
+    });
+
+    <% if (user.admin) { %>
+    $deleteBtn = $('#deleteBtn');
+    $editBtn = $('#editBtn');
+
+    $table.on('check.bs.table uncheck.bs.table check-all.bs.table uncheck-all.bs.table', function () {
+        $deleteBtn.prop('disabled', !$table.bootstrapTable('getSelections').length);
+        $editBtn.prop('disabled', $table.bootstrapTable('getSelections').length != 1);
+    });
+
+    $deleteBtn.click(function () {
+        var r = confirm("Are you sure you want to delete the selected projects");
+        if (r == false) {
+            return;
+        }
+
+        var data = {};
+        data.id = getIdSelections();
+
+        $.ajax({
+            type: "DELETE",
+            url: '/api/project',
+            contentType: "application/json",
+            data: JSON.stringify(data),
+            complete: function (result) {
+                if (result.success) {
+                    $table.bootstrapTable('refresh');
+                    $deleteBtn.prop('disabled', true);
+                } else {
+                    alert("There was an error deleting technology :(");
+                }
+            }
+        });
+
+        <% if (google_id!=undefined) { %>
+        ga('send', 'event', 'Project', 'delete', data.id);
```

Comment:
@xjedam  How about this event? Does it do the job?
https://github.com/KainosSoftwareLtd/TechRadar/blob/master/views/partials/head.ejs#L20